tests/View/Mix/MixTest.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+namespace Illuminate\Tests\View\Mix;
+
+use PHPUnit\Framework\TestCase;
+use Illuminate\Support\HtmlString;
+
+class MixTest extends TestCase
+{
+    public function setUp()
+    {
+        app()->singleton('path.public', function () {
```

Comment:
The other JSON files (valid and not valid) are fixtures and should be committed.

It does matter - the empty HMR files should not; see `\Illuminate\Tests\Filesystem\FilesystemTest::setUp` so that we don't have to commit empty folders in tests.